### PR TITLE
testing travis-ci on osx

### PR DESCRIPTION
**Pull request by luzpaz**
this is just a test. I'm using travis-ci to determine if it works.

### CODE REVIEW

**Comment by luzpaz** [comment]
Yikes, that took a while. 
Ok so OSX now builds correctly on Travis. 
Next step is to integrate OSX build in to the current travis.yml file. 
This [gist](https://gist.github.com/luzpaz/82ab8a0dad9e4f5520f1) is where that is being done.
Once that is done I'll post the .yml file to bugs.scribus.net

**Comment by luzpaz** [comment]
@QuLogic If you have a moment, can you offer any help to have Linux and OSX play well together in https://gist.github.com/luzpaz/82ab8a0dad9e4f5520f1 
The idea is to have linux built with both gcc and clang. And for OSX to just be built w/ clang.

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,53 @@
-language: cpp
```
Comment (OceanWolf) [inline .travis.yml]: Why remove this?

File: .travis.yml [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
+#    - os: linux
+#      dist: trusty
+#      sudo: required
+    - os: osx
 
-compiler:
-  - gcc
-  - clang
-
-dist: trusty
```
Comment (OceanWolf) [inline .travis.yml]: And this?

File: .travis.yml (hunk 1 above)
Comment (OceanWolf) [inline .travis.yml]: Oh, you moved this down below, hmm...

**Comment by luzpaz** [comment]
@OceanWolf this PR is just a proof of concept. I didn't know that Travis-ci took OSX out of of invite-only beta test trials (scribusproject/scribus is part of those trials) and I didn't sign up my own repo for it. Long story short was using this repo to trigger travis-ci tests to get OSX to work. The travis.yml script that is moving toward the direct of multiple OS's is at this gist: https://gist.github.com/luzpaz/82ab8a0dad9e4f5520f1

**Comment by luzpaz** [comment]
There is a problem with the yaml markup. 
```syntax error: (<unknown>): mapping values are not allowed in this context at line 5 column 11```
working on it

**Comment by luzpaz** [comment]
OK, milestone reached. All 3 builds show up!
1. Linux GCC
2. Linux Clang
3. OSX Clang
What's left is to fix the reason why 1 and 2 are failing at ```make install```
Error: https://travis-ci.org/scribusproject/scribus/jobs/84433814#L4537
```
Install the project...
-- Install configuration: "RELEASE"
CMake Error at cmake_install.cmake:36 (FILE):
file cannot create directory: /usr/local/share/doc/scribus.  Maybe need administrative privileges.
```
What's confusing is that sudo is enabled for the Linux builds. So I'm not sure what needs to be tweeked.

**Comment by luzpaz** [comment]
submitted to mantis bugtracker http://bugs.scribus.net/view.php?id=13472

**Comment by luzpaz** [comment]
Patch was committed in r20505 by craig. Closing

**Pull request closed without merging**